docs/v0.3/sdk/go/xfgss-faq.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: /xfgss/faq
+---
+
+# FAQ
```

Comment:
Please review these questions and answers and let me know if we need to add or update them. 

Note that this FAQ is presented at the root level in the navigation tree and not under the "Dagger SDK for Go" node because the questions we have to date seem to be more about Dagger 0.3 than specifically about the Go SDK.